Assets/Scripts/UI/InGameUI/DevConsole/DevConsole.cs

```diff
@@ -378,7 +456,7 @@ public static void ShowHelpMethod(CommandBase help)
             }
 
             Log("<color=yellow>Command Info:</color> " + (string.IsNullOrEmpty(help.DetailedDescription) ? help.Description : help.DetailedDescription));
-            Log("<color=yellow>Call it like </color><color=orange> " + help.Title + GetParameters(help) + "</color>");
+            Log("<color=yellow>Call it like </color><color=orange> " + help.Title + GetParameters(help));
```

Comment:
Is there a reason you no longer need to close out the color tag?

Reply:
It technically is fine since GetParameters now controls its own colour (rather than it used to just output the text raw it now colours it so it can colour the types separately than the names so it is clearer to read) there are a few cases where this could result in colours leaking so I do actually need to close out the colour tag.

Submitted fix just now.